provider/vsphere/internal/vsphereclient/createvm_test.go

```diff
@@ -384,6 +385,35 @@ func (s *clientSuite) TestCreateVirtualMachineRootDiskSize(c *gc.C) {
 	})
 }
 
+func (s *clientSuite) TestVerifyMAC(c *gc.C) {
+	var testData = []struct {
+		Mac    string
+		Result bool
+	}{
+		{"foo:bar:baz", false},
```

Comment:
If we're going with a fixed set of cases here, let's validate all the edges. For instance I notice we're not playing with mac that have letters outside of a-f, nor having letters outside the right locations, nor going over the max value, etc.